rfcs/text/0003-web-rpc-hooks.md

```diff
@@ -0,0 +1,280 @@
+* Start Date: 2019-05-28 
+* RFC PR: (leave this empty)
+* Fusion Issue: (leave this empty)
+
+# Summary
+
+This PR documents a suggested path for supporting hooks in web-rpc, as well as options
+for suspense compatible APIs.
+
+# Basic example
+
+NOTE: All of these examples assume a parent Suspense and ErrorBoundary, as well as a corresponding reducer for user actions.
```

Comment:
I think it'd be worth showing the JSX of what the Suspense + ErrorBoundary would look like.